nservicebus/rabbitmq/delayed-delivery.md

```diff
@@ -108,22 +108,22 @@ end
 ```
 
 
-## Settings
+## Backwards compatibility
+
+When using a version of the transport that supports delayed delivery natively, the endpoint continues to operate in a backwards-compatible manner by default. The timeout manager will continue to be enabled, so any delayed messages already stored in the endpoint's persistence database will be sent when their timeouts expire. New delayed messages will be sent through the delay infrastructure, but the endpoint cannot assume that other endpoints have also been upgraded, so it will incur some additional overhead when sending delayed messages to ensure that they can be delivered successfully.
```

Comment:
I prefer the sentence how it currently is, so let's see if we get any additional feedback on it first.